ppml/base/encryptedfsd.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
change this script to encrypted-fsd.sh ?

Reply:
Fixed